tests/ballerina-test/src/test/resources/test-src/connectors/sql/sql_actions_test.bal

```diff
@@ -816,6 +816,52 @@ function testBatchUpdate() returns (int[]) {
     return updateCount;
 }
 
+type myBatchType string|int|float;
+
+function testBatchUpdateWithValues() returns (int[]) {
+    endpoint jdbc:Client testDB {
+        url: "jdbc:hsqldb:file:./target/tempdb/TEST_SQL_CONNECTOR",
+        username: "SA",
+        poolOptions: { maximumPoolSize: 1 }
+    };
+
+    //Batch 1
+    myBatchType[] parameters1 = ["Alex", "Smith", 20, 3400.5, "Colombo"];
+
+    //Batch 2
+    myBatchType[] parameters2 = ["John", "Gates", 45, 2400.5, "NY"];
+
+    int[] updateCount = check testDB->batchUpdate("Insert into Customers (firstName,lastName,registrationID,
+                            creditLimit,country) values (?,?,?,?,?)", parameters1, parameters2);
+    testDB.stop();
+    return updateCount;
+}
+
+function testBatchUpdateWithVariables() returns (int[]) {
```

Comment:
Shall we remove brackets in the return val because brackets are specifically used for tuple types.